memory/migrate_pages.py.data/node_move_pages.c

```diff
@@ -328,17 +330,26 @@ int main(int argc, char *argv[])
 			chunks = strtoul(optarg, NULL, 10);
                         break;
                 case 'o':
+#ifdef HAVE_HUGETLB_HEADER
 			overcommit = 1;
-                        break;
-                case 't':
+#else			
+                        overcommit = 0;
+#endif			
+			break;
+
+		case 't':
 			thp = 1;
                         break;
                 case 'h':
+#ifdef HAVE_HUGETLB_HEADER
 			hugepage = 1;
 			pagesize = gethugepagesize();
 			/* Using 1% of system memory for hugepages*/
         		memory = (total_mem) / 100;
-                        break;
+#else
+			hugepage = 0 ;
+#endif		       	
+			break;
                 default:
                         errmsg("%s [-n <no-of-chunks] [-t fot THP] [-o for hugepage-overcommit] [-h for hugepage]\n", argv[0]);
```

Comment:
add a if .. else here to update the errmsg.For the case without hugetlb headers remove the -o and -h option from this message.